pkgs/applications/emulators/wine/update-lib.sh

```diff
@@ -0,0 +1,50 @@
+sources_file=pkgs/applications/emulators/wine/sources.nix
```

Comment:
Please do not hardcode the file path here

Reply:
Is hardcoding it somewhere else (in sources.nix) okay? Or should I try to do something clever with the secret `__curPos` builtin or [something like that](https://github.com/NixOS/nix/issues/5897)?

Reply:
Went with `__curPos`; LMK if something else is desired.